[33mWARNING [0m urllib3.connectionpool:connectionpool.py:744 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000004C0BC70>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/c8ff2160eafa209597aa5202ea87eede/url
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:744 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000004C0BDF0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/c8ff2160eafa209597aa5202ea87eede/url
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:744 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000004C0B070>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/c8ff2160eafa209597aa5202ea87eede/url